heat/core/tests/test_rounding.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import heat as ht
 
+from heat.core.tests.test_suites.basic_test import BasicTest
+
 
 class TestRounding(unittest.TestCase):
```

Comment:
The BasicTest class is meant as an extension to the `unittest.TestCase` class and should therefore be extended when you want to use it.

Reply:
Thank you for your feedback! I tried to implement your requested changes.